scalajslib/src/ScalaJSModule.scala

```diff
@@ -197,8 +197,22 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
 
   def moduleKind: T[ModuleKind] = T { ModuleKind.NoModule }
 
-  def useECMAScript2015: T[Boolean] = T { 
-    !scalaJSVersion().startsWith("0.")
+  @deprecated("Use esFeatures().esVersion instead", since = "mill after 0.10.0-M5")
+  def useECMAScript2015: T[Boolean] = T {
+    esFeatures().esVersion != ESVersion.ES5_1
```

Comment:
I would rather just keep the old default, so we can depend on it in `esFeatures`.

Reply:
Done ✅